SerializingTest/GraphConversionTest.cs

```diff
@@ -0,0 +1,76 @@
+using NUnit.Framework;
+using CraphModel;
+using System.Collections.Generic;
+using Serializing;
+
+
+
+namespace SerializingTest
+{
+    public class GraphConversionTest
+    {
+
+
+        private Graph graph;
+        private Vertex vertex;
+        private SerializeGraph serializeGraph;
+        private DeserializeGraph deserializeGraph;
+       
+
+        [SetUp]
+        public void Setup()
+        {
+            graph = new Graph();
+            vertex = new Vertex();
+            serializeGraph = new SerializeGraph();
+            deserializeGraph = new DeserializeGraph();
+
+            graph.Vertexs = new List<Vertex>(1);
+            vertex.Nodes = new List<Node>(2);
+
+            vertex.Nodes.Add(new Node() { Weight = 228, Connectable = 20 });
+            vertex.Nodes.Add(new Node() { Weight = 14, Connectable = 17 });
+
+            graph.Vertexs.Add(new Vertex() { Nodes = vertex.Nodes, Id = 42 });
+
+        }
+
+        [Test]
+        public void SerializationEdgeWeight_Weight_228return()
```

Comment:
After the name of the test method, the word "Test"should be written. Example: SomeMethodTest_Scenario_ExpectedResult